docker/Dockerfile

```diff
@@ -0,0 +1,73 @@
+# This Dockerfile creates a production release image for the project. This
+# downloads the release from releases.hashicorp.com and therefore requires that
+# the release is published before building the Docker image.
+#
+# We don't rebuild the software because we want the exact checksums and
+# binary signatures to match the software and our builds aren't fully
+# reproducible currently.
+ARG BASE_ARCH
+FROM ${BASE_ARCH}/alpine:3.11
```

Comment:
```suggestion
FROM alpine:3.13
```

Reply:
Just pushed the commit that changes this.

Reply:
Thanks!